plugins/scoreboard.js

```diff
@@ -86,41 +87,57 @@ function getEntitiesCount (entities,bounds) {
   return {
     enl: counts[window.TEAM_ENL] || 0,
     res: counts[window.TEAM_RES] || 0,
+    mac: counts[window.TEAM_MAC] || 0,
   };
 }
 
 function makeTable (portals,linksCount,fieldsCount) {
 
   var html = '';
   html += '<table>'
-  + '<colgroup><col><col class="enl"><col class="res"></colgroup>'
+  + '<colgroup><col><col class="enl"><col class="res"><col class="mac"></colgroup>'
   + '<tr>'
   + '<th>Metrics</th>'
   + '<th class="enl">Enlightened</th>'
   + '<th class="res">Resistance</th>'
+  + '<th class="mac">__MACHINA__</th>'
   + '</tr>\n';
 
   html += '<tr><td>Portals</td>'
     +'<td>'+portals.enl.total+'</td>'
-    +'<td>'+portals.res.total+'</td></tr>'
+    +'<td>'+portals.res.total+'</td>'
+    +'<td>'+portals.mac.total+'</td>'
+    +'</tr>'
   +'<tr><td>avg Level</td>'
     +'<td>'+portals.enl.levels+'</td>'
-    +'<td>'+portals.res.levels+'</td></tr>'
+    +'<td>'+portals.res.levels+'</td>'
+    +'<td>'+portals.mac.levels+'</td>'
+    +'</tr>'
   + '<tr><td>avg Health</td>'
     +'<td>'+portals.enl.health+'</td>'
-    +'<td>'+portals.res.health+'</td></tr>'
+    +'<td>'+portals.res.health+'</td>'
+    +'<td>'+portals.mac.health+'</td>'
+    +'</tr>'
   +'<tr><td>Level 8</td>'
     +'<td>'+portals.enl.level8+'</td>'
-    +'<td>'+portals.res.level8+'</td></tr>'
+    +'<td>'+portals.res.level8+'</td>'
+    +'<td>'+portals.mac.level8+'</td>'
+    +'</tr>'
   +'<tr><td>Max Level</td>'
     +'<td>'+portals.enl.maxLevel+'</td>'
-    +'<td>'+portals.res.maxLevel+'</td></tr>'
+    +'<td>'+portals.res.maxLevel+'</td>'
+    +'<td>'+portals.mac.maxLevel+'</td>'
+    +'</tr>'
   +'<tr><td>Links</td>'
     +'<td>'+linksCount.enl+'</td>'
-    +'<td>'+linksCount.res+'</td></tr>'
+    +'<td>'+linksCount.res+'</td>'
+    +'<td>'+linksCount.mac+'</td>'
+    +'</tr>'
   +'<tr><td>Fields</td>'
     +'<td>'+fieldsCount.enl+'</td>'
-    +'<td>'+fieldsCount.res+'</td></tr>';
+    +'<td>'+fieldsCount.res+'</td>'
```

Comment:
Imho this whole table could be better rendered using some loops. 
I know it was like this for years... but if its not too much trouble, could you rewrite it that way?

Just if you feel up to it and have time for it
Otherwise your changes are as good as the original so... 👍